mathesar_ui/src/component-library/with-view-more-or-less-button/WithViewMoreOrLessButton.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+  export let content: string;
+  export let maxLength: number;
```

Comment:
I'm curious as to why we are implementing this in js instead of css. This will result in the view looking differently with different text & languages.

A better option would be to truncate based on number of lines:
* Get `maxLines` as a prop.
* Use [webkit-line-clamp](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp) to truncate multi-line text using the `maxLines` prop. We currently use this for displaying multi-line text when a cell becomes active.
   - There are issues with this property, but it'll ensure the UI doesn't look different based on different text & languages.

This isn't needed for the live demo and no change is needed on this PR. We may need it for the release with localization. Please create an issue to track this.

Reply:
The new design in Figma uses a tooltip instead of a "view more" button, so it might be easiest to remove this feature for now.

Reply:
Got it removing this component completely for now